content/reference/important-protocols.adoc

```diff
@@ -24,12 +24,16 @@ Routers use this when building routes: anything that satisfies
 `IntoInterceptor` is legal to use in the interceptor vector of a
 route.
 
-As covered in the
-link:interceptors#_manipulating_the_interceptor_queue[Interceptors]
-reference, interceptors that are explicitely enqueued by the
-application must be defined using the
-`io.pedestal.interceptor/interceptor` function. Queue processing is
-optimized for runtime and does not construct `Interceptor` records.
+[IMPORTANT]
+.*Dynamically added interceptors*
+
+--
+At runtime, new interceptors may be added to the
+link:interceptors#_manipulating_the_interceptor_queue[queue] dynamically;
+(this is what routing itself does); to ensure best runtime performance, `IntoInterceptor` is not
+automatically used; it is assumed any added interceptors are, in fact,
```

Comment:
I don't think I would suggest it's "assumed" here.  In most cases, Pedestal checks chains (and routes) and ensures the IntoIntercator transformation is applied before the service if fully running.

Reply:
I've converted this to largely just link to the details on queuing interceptors.  Reads better.